### ISSUE TITLE
Resetting the TPM emulator may be neccesary

### ISSUE THREAD

**Issue by lkatalin** [keylime/rust-keylime#174]
```
[root@localhost ~]# ps aux | grep swtpm
root        5693  0.0  0.1   7104  3600 ?        Ss   15:26   0:00 swtpm chardev --tpm2 --vtpm-proxy --tpmstate dir=/var/vtpm -d
root        5750  0.0  0.0 221432   852 pts/0    S+   15:30   0:00 grep --color=auto swtpm
[root@localhost ~]# kill 5693
[root@localhost ~]# ./swtpm.sh 
Found existing /var/vtpm
New TPM device: /dev/tpm1 (major/minor = 253/1)
[root@localhost ~]# source swtpm.sh
Found existing /var/vtpm
New TPM device: /dev/tpm2 (major/minor = 253/2)
[root@localhost ~]# echo $TPM2TOOLS_TCTI
device:/dev/tpmrm0
[root@localhost ~]# keylime_ima_emulator
Using config file /etc/keylime.conf
2021-03-24 15:31:11.227 - keylime.tpm - INFO - TPM2-TOOLS Version: 4.3.0
Warning: IMA PCR is not empty, trying to find the last updated file in the measurement list...
Traceback (most recent call last):
  File "/usr/local/bin/keylime_ima_emulator", line 33, in <module>
    sys.exit(load_entry_point('keylime==0.0.0', 'console_scripts', 'keylime_ima_emulator')())
  File "/usr/local/lib/python3.9/site-packages/keylime-0.0.0-py3.9.egg/keylime/cmd/ima_emulator_adapter.py", line 87, in main
    pos = ml_extend(config.IMA_ML, 0, pcrval)
  File "/usr/local/lib/python3.9/site-packages/keylime-0.0.0-py3.9.egg/keylime/cmd/ima_emulator_adapter.py", line 70, in ml_extend
    raise Exception(
Exception: Unable to find current measurement list position, Resetting the TPM emulator may be neccesary
```

**Comment by lkatalin** [comment]
Instead of `source swtpm.sh`, manually export `TPM2TOOLS_TCTI`:
```
[root@localhost ~]# ./swtpm.sh 
Found existing /var/vtpm
New TPM device: /dev/tpm1 (major/minor = 253/1)
[root@localhost ~]# export TPM2TOOLS_TCTI="device:/dev/tpm1"
[root@localhost ~]# keylime_ima_emulator
Using config file /etc/keylime.conf
...
```

Context (`TPM2TOOLS_TCTI` var was not updated):
```
[root@localhost ~]# ./swtpm.sh 
Found existing /var/vtpm
New TPM device: /dev/tpm1 (major/minor = 253/1)
[root@localhost ~]# source swtpm.sh
Found existing /var/vtpm
New TPM device: /dev/tpm2 (major/minor = 253/2)
[root@localhost ~]# echo $TPM2TOOLS_TCTI
device:/dev/tpmrm0
[root@localhost ~]# export TPM2TOOLS_TCTI="device:/dev/tpm1"
[root@localhost ~]# keylime_ima_emulator
Using config file /etc/keylime.conf
...
```

**Comment by lkatalin** [comment]
tl;dr this seems to occur when the TCTI is not the correct tpm number
